Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -163,6 +163,10 @@ open class Player: BaseObject {
     open func seek(_ timeInterval: TimeInterval) {
         core?.activePlayback?.seek(timeInterval)
     }
+    
+    open func mute(enabled: Bool) {
```

Comment:
Hey @torives, maybe we can expose two methods and split into unique responsibility. What do you think?